views/error.html

```diff
@@ -1,8 +1,11 @@
-<h1 class="base--h1">I'm error!</h1>
+<div class="not-found">
```

Comment:
Because I wasn't sure of the reasoning of a separate 404 vs error view. Although both using the _exact same_ template now, I wanted to leave 404 on it's own so that any changes to a 404 page in the future would not affect the general error page.